runbotics/runbotics-desktop/src/utils/botSystem.ts

```diff
@@ -0,0 +1,15 @@
+import { BotSystem } from 'runbotics-common';
+
+const getBotSystem = () => {
+    const system = process.platform;
+    switch (system) {
+        case 'win32':
+            return BotSystem.WINDOWS;
+        case 'linux':
+            return BotSystem.LINUX;
+        default:
+            return system.toUpperCase();
```

Comment:
why are we returning this? if it has no use, imo it's better to throw error

Reply:
I think that this is not a good place to throw an error, considering that this function only reads and maps values from process.platform. Also I am throwing exception in validateBotSystem in auth.service.ts

Reply:
alright